pdf/core/utils.go

```diff
@@ -38,6 +38,18 @@ func getUniDocVersion() string {
 	return common.Version
 }
 
+// GetObjectNums returns the object numbers of the PDF objects in the file
+func (this *PdfParser) GetObjectNums() (map[int]int, error) {
```

Comment:
I have added the function
`func (r *PdfReader) GetObjectNums() []int {`
and merged, updated the example pdf_all_objects.go to use it.